vm.go

```diff
@@ -351,6 +353,51 @@ func (vm *VirtualMachineSpec) ExecuteCommand(command string) error {
 	return nil
 }
 
+// RunCommand executes a command on the Virtual Machine
+func (vm *VirtualMachineSpec) RunCommand(command string) (output string, err error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Minute)
```

Comment:
thought about modifying ExecuteCommand API which isn't implemented, but chose to go with a new API since that is public and needs signature change. @johnsonshih , your thoughts?